configureWorkspace/configure_go.ts

```diff
@@ -3,15 +3,18 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { PackageInfo } from './configure';
+import { getExposeStatements, IPlatformGeneratorInfo, PackageInfo } from './configure';
 
-export let configureGo = {
+export let configureGo: IPlatformGeneratorInfo = {
   genDockerFile,
   genDockerCompose,
-  genDockerComposeDebug
+  genDockerComposeDebug,
+  defaultPort: '3000'
 };
 
 function genDockerFile(serviceNameAndRelativePath: string, platform: string, os: string | undefined, port: string, { cmd, author, version, artifactName }: Partial<PackageInfo>): string {
+  let exposeStatements = getExposeStatements(port);
```

Comment:
There seems to be an inconsistency where sometimes the `genDockerFile` has a 'port' parameter and sometimes it has an 'exposeStatements' parameter. Can we make that consistent?

Reply:
done